contrib/docker/Dockerfile

```diff
@@ -7,9 +7,10 @@ COPY rootfs_prefix/ /usr/src/rootfs_prefix/
 
 RUN apt-get update \
  && apt-get upgrade \
- && apt-get install \
+ && DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends -y \
```

Comment:
Hi!

1) `DEBIAN_FRONTEND=noninteractive` already set in line 3.
2) `--no-install-recommends` and  `-y` already set in `50docker-apt-conf`.

Please remove these changes.

Thanks for your contribution!

Reply:
I see.
If you have such a policy, I think it is good to write so.
```diff
--- a/contrib/docker/Dockerfile
+++ b/contrib/docker/Dockerfile
@@ -11,6 +11,8 @@ RUN apt-get update \
     collectd-core \
     collectd-utils \
     build-essential \
+ && : "If you need some shared libraries when enabling plugins please add the package here." \
+ && : "FYI. https://github.com/collectd/collectd/pull/2974" \
  && make -C /usr/src/rootfs_prefix/ \
  && apt-get --purge remove build-essential \
  && apt-get clean \
```